src/nvim/search.c.rej

```diff
@@ -0,0 +1,662 @@
+--- src/nvim/search.c
```

Comment:
Remove this file.

Reply:
File has been removed and lint error fixed. The tests are not passing for some reason though. Seems related to `TV_LIST_ITER, that I don't really know how to use.